Make tabs on memory screen looking nicer. (#4291)
flutter/devtools: packages/devtools_app/lib/src/screens/memory/panes/README.md

Panes in this folder should not depend on each other.

> If they need to share code, move the code to memory/primitives (if it is one library or less),
> or to memory/shared (if it is more than one library).

Reviewer: How can it be in one library OR LESS?

```suggestion
If any panes need to share code, move the code to memory
or primitives if used in one library,
or to memory or shared
if used in more than one library.
```

Author: memory/primitives is a path that consists of two folders.  Added ticks.
